Client/Frontend/Strings.swift

```diff
@@ -280,6 +280,12 @@ extension Strings {
     public static let ReaderModeResetFontSizeAccessibilityLabel = NSLocalizedString("Reset text size", comment: "Accessibility label for button resetting font size in display settings of reader mode")
 }
 
+// Leanplum Strings.
+extension Strings {
```

Comment:
I don't think we need these String, right? Everything comes from LP so we don't need to localize anything?